integr8ly/integreatly-operator
# MGDAPI-4813 Update RHSSO operator to 7.6.1-3

# Issue link
JIRA: https://issues.redhat.com/browse/MGDAPI-4813

# What
Updates RHSSO operator from `7.6.0-8` -> `7.6.0-10` -> `7.6.1-3`

# Verification steps
1. Provision cluster
2. Checkout rhoam-v1.28.0: `git checkout tags/rhoam-v1.28.0`
3. Build RHOAM operator, bundle, and index images for rhoam-v1.28.0 and push them to your quay org
4. Prepare the cluster for the installation: `INSTALLATION_TYPE=managed-api LOCAL=false make cluster/prepare/local`
5. Install RHOAM via OperatorHub (OLM install)
6. Create the RHMI CR: `INSTALLATION_TYPE=managed-api USE_CLUSTER_STORAGE=true LOCAL=false IN_PROW=true make deploy/integreatly-rhmi-cr.yml`
7. Wait for the installation to complete
8. Check out this PR
9. Build RHOAM operator, bundle, and index images for rhoam-v1.29.0 and push them to your quay org
10. Update the CatalogSource to point to rhoam-v1.29.0
11. Wait for the upgrade to complete
12. Confirm that the RHSSO operator has been updated to `7.6.1-3` in both the `redhat-rhoam-rhsso-operator` and `redhat-rhoam-user-sso-operator` Namespaces
